src/osd/PG.cc

```diff
@@ -5944,7 +5944,7 @@ boost::statechart::result PG::RecoveryState::Peering::react(const AdvMap& advmap
     post_event(advmap);
     return transit< Reset >();
   }
-  
+  pg->check_full_transition(advmap.lastmap, advmap.osdmap);
```

Comment:
The Peering and Active states are both sub-states of Started, which means the check should already get handled there, right?